speech/grpc/src/main/java/com/google/cloud/speech/grpc/demos/AsyncRecognizeClient.java

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
Please run google-java-format against this file: https://github.com/google/google-java-format

I'm finding it slightly difficult to read.